src/components/global/Header/styles.js

```diff
@@ -0,0 +1,9 @@
+import { createUseStyles } from "react-jss";
+
+const useStyles = createUseStyles({
+  menuIcon: {
+    margin: "1.5rem",
```

Comment:
use rem function directly, and not explicitly a rem unit so it has more consistency